### ISSUE TITLE
Drop comma after the last Enum value

### ISSUE THREAD

**Issue by Maximaximum** [reinforced/Reinforced.Typings#67]
I'm not sure if this feature is within the scope of the project, but it would be great if we could drop the comma after the last Enum value:
```
export enum TaxType {
  NoTax = 0,
  SomeTax = 1,
  MuchTax = 2,
}
```
should be
```
export enum TaxType {
  NoTax = 0,
  SomeTax = 1,
  MuchTax = 2
}
```

Formatters like `typescript-formatter` don't seem to have an option to remove the trailing commas, so maybe this should be handled by the emitter?

**Comment by pavel-b-novikov** [comment]
I wil wait for a while until you collect more issues

**Comment by pavel-b-novikov** [comment]
Fixed in 1.4.93
